frontend/src/employee/MoEmployeeWorkflows/MoEmployeeCreate.vue

```diff
@@ -138,7 +139,11 @@ export default {
      * Create a employee and check if the data fields are valid.
      * Then throw a error if not.
      */
+    setOrganisation () {
+      this.organisation = this.$store.getters['organisation/get']
```

Comment:
You could make this into a computed property instead. Import `mapGetters` and do something like
```
...mapGetters({
organisation: 'organisation/get'
})

Reply:
Neat. I'll try that.